packages/foreman/keycloak-httpd-client-install/keycloak-httpd-client-install.spec

```
@@ -2,18 +2,29 @@
 %global pypi_name keycloak-httpd-client-install
 %global summary Tools to configure Apache HTTPD as Keycloak client
 
+%if 0%{?rhel} == 7
+%global with_python2 1
+%else
+%global with_python3 1
+%endif
+
 Name:           %{pypi_name}
 Version:        1.2.2
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        %{summary}
 
 License:        GPLv3
 URL:            https://github.com/latchset/keycloak-httpd-client-install
 Source0:        https://files.pythonhosted.org/packages/source/k/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
 BuildArch:      noarch
- 
+
+%if 0%{?rhel} == 7
 BuildRequires:  python2-devel
 BuildRequires:  python2-setuptools
+%else
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+%endif
```

```suggestion
%if %{with python2}
BuildRequires:  python2-devel
BuildRequires:  python2-setuptools
%endif

%if %{with python3}
BuildRequires:  python3-devel
BuildRequires:  python3-setuptools
%endif
```

so technically you can build both in parallel